### PR TITLE
Create promotion page and display promotion at homepage

### PR DESCRIPTION
**Pull request by truongprohk2502**
https://edu-redmine.sun-asterisk.vn/issues/35727

### CODE REVIEW

**Comment by truongprohk2502** [comment]
ready

**Comment by truongprohk2502** [comment]
ready

File: src/components/additional-film-list/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./style.sass";
+import { useTranslation } from "react-i18next";
+
+function AdditionalFilmList(props) {
```
Comment (manhnt-0960) [inline src/components/additional-film-list/index.js]:
```suggestion
function AdditionalFilmList({movies}) {
```

File: src/components/additional-film-list/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <div className="sub-content-title">
+        {t("common.list_title.playing_movie")}
+      </div>
+      {props.movies &&
+        props.movies.map((movie) => (
```
Comment (manhnt-0960) [inline src/components/additional-film-list/index.js]: movies.length && movies.map

**Comment by truongprohk2502** [comment]
ready

**Pull request merged**